pulpcore/openapi/__init__.py

```diff
@@ -258,6 +258,16 @@ def resolve_serializer(self, serializer, direction):
 
         return component
 
+    def _get_response_bodies(self):
+        """
+        Handle response status code.
+        """
+        response = super()._get_response_bodies()
+        if self.method == "POST" and issubclass(self.view.__class__, mixins.CreateModelMixin):
+            return dict(("201", value) for (key, value) in response.items())
```

Comment:
Wow, this is brutal. How do you think about this:
```suggestion
        if self.method == "POST" and issubclass(self.view.__class__, mixins.CreateModelMixin) and "201" not in response:
            response["201"] = response["200"]
```